src/content/security/index.md

```diff
@@ -232,6 +232,20 @@ These fraudulent brokers find their targets by using fake accounts on YouTube to
 
 ![A trading broker scam on YouTube](./brokerScam.png)
 
+### Crypto mining pool scams {#mining-pool-scams}
+
+Mining pool scams involve people contacting you unsolicited, and claiming that you can make large returns by joining an Ethereum mining pool. The scammer will make claims and stay in contact with you for however long it takes. Essentially, the scammer will try and convince you that when you join an Ethereum mining pool, your cryptocurrency will be used to create Ethereum and that you will be paid dividends in the form of Ethereum. What ends up happening is, you will notice that your cryptocurrency is making small returns. This is simply to bait you into investing more. Eventually, all of your funds will be sent to an unknown address and the scammer will either disappear or in some cases will continue to stay in touch as has happened in a recent case.
```

Comment:
```suggestion
Mining pool scams involve people contacting you unsolicited, and claiming that you can make large returns by joining an Ethereum mining pool. The scammer will make claims and stay in contact with you for however long it takes. Essentially, the scammer will try and convince you that when you join an Ethereum mining pool, your cryptocurrency will be used to create Eth and that you will be paid dividends in the form of Eth. What will end up happening is, you will notice that your cryptocurrency is making small returns. This is simply to bait you into investing more. Eventually, all of your funds will be sent to an unknown address and the scammer will either disappear or in some cases will continue to stay in touch as has happened in a recent case.
```

Reply:
just thought there should be a distinction between Ethereum the blockchain and Eth the asset.

Reply:
Hi @nassersaazi I think you are right, thanks! Would you mind adding a commit that changes Eth to ETH please first?

Reply:
alright